css/style.css

```diff
@@ -0,0 +1,82 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body{
+    background-color: #eee;
+    color: #333;
+    font-size: 1.3rem;
+    line-height: 1.5;
+}
+h1, h2{
+    padding: 0.5rem 0;
+}
+p{
+    padding: 0.3rem 0;
+}
+
+#nav-bar{
+    padding: 2rem;
+    display: flex;
+    justify-content: space-between;
+
+}
+ul{
+    display: flex;
+}
+li{
+    list-style-type: none;
+}
+a{
+    text-decoration: none;
+    color: #333;
+    padding: 0 1rem;
+}
+img{
+    width: 200px;
+}
+#header{
+text-align: center;
+padding: 1rem 0;
+}
+#form{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  
+}
+input{
+    padding: 0.4rem;
+}
+#submit{
+    margin: 1rem 0;
+}
+.container{
+    width: 60%;
+    margin: auto;
+}
+#video{
+    margin: 2rem 0;
+    text-align: center;
+}
+.footer{
```

Comment:
[OPTIONAL] Just target the footer

Reply:
> [OPTIONAL] Just target the footer

Kindly suggest changes politely and includes an explanation for your requested change. You can follow [these steps](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) to create an effective review.